packages/siri-sx-validator/.gitignore

```diff
@@ -0,0 +1,6 @@
+*
+
+!.gitignore
+!index.py
+!requirements.txt
+!scripts
```

Comment:
Do we need to allow the xsd folder here as well?

Reply:
xsd added